atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -411,7 +412,8 @@ private void saveFinalSweepResults(TableToSweep tableToSweep, SweepResults sweep
             return null;
         });
 
-        sweepMetrics.recordMetrics(tableToSweep.getTableRef(), sweepResults);
+        sweepMetrics.examinedCells(tableToSweep.getTableRef(), sweepResults.getCellTsPairsExamined());
```

Comment:
I see that there's been a shift in responsibility here, in that for happy-case metrics, it's now the responsibility of `BackgroundSweeperImpl` rather than `SweepMetrics` to know which metrics to record at which time.

Are we planning to add more happy-case metrics, and if so, would they be recorded here or elsewhere?